Documentation/Helm-Charts/ceph-cluster-chart.md

```diff
@@ -86,6 +86,17 @@ The following table lists the configurable parameters of the rook-operator chart
 
 The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
 For the full list, see the [Cluster CRD](../CRDs/Cluster/ceph-cluster-crd.md) topic.
+!!! Note
```

Comment:
I'm not sure if it's critical to the rendering, but everywhere we have lower case `note`, and good to have a blank line before it.
```suggestion

!!! note
```